### ISSUE TITLE
expected results on CIFAR

### ISSUE THREAD

**Issue by hmichaeli** [facebookresearch/ConvNeXt#58]
Can you please share the expected results or training regime for CIFAR100 dataset (if exist)?
Using the same regime posted for conext-tiny on ImageNet yields test_acc1 of 82.1 on CIFAR100. It seems quite low compared to Imagenet results.

specifically I ran the script the following:

python -m torch.distributed.launch --nproc_per_node=8 main.py \
--model $ARCH --drop_path 0.1 \
--batch_size 64 --lr 4e-3 --update_freq 8 \
--model_ema true --model_ema_eval true \
--data_set CIFAR \
--data_path $CIFAR_DIR \
--output_dir $ROOT_RES \
--seed $SEED

**Comment by liuzhuang13** [comment]
Hi,

We don't have expected results for CIFAR datasets. If you desire reasonable CIFAR results, my own suggestion is to migrate ConvNeXt model files to codebases for training on CIFAR and use their training hyper-parameters (despite that ConvNeXt is 4-stage and typical CIFAR models are 3-stage I believe, so there is room for adapting the model too). Our training script inherits from those designed for ImageNet data only, so an ideal CIFAR recipe might be very different

**Comment by jiyuanyu** [comment]
您好，请问我想测试自己的数据集，应该怎么修改代码？
